uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-fs/src/main/java/org/uberfire/java/nio/fs/file/SimpleFileSystemProvider.java

```diff
@@ -133,8 +129,7 @@ public Path getPath(final URI uri)
 
     @Override
     public FileSystem newFileSystem(final URI uri,
-                                    final Map<String, ?> env)
-            throws IllegalArgumentException, IOException, SecurityException, FileSystemAlreadyExistsException {
+                                    final Map<String, ?> env) {
```

Comment:
why not preserve the original throws? this was more for `documentation` purposes

Reply:
Well, I do 100% agree with you, but the sonarCloud flags it as a rule violation.

Reply:
Reverted back to original style, but SonarCloud might not be happy :(